pages/national_resources/cy_resources.md

```diff
@@ -53,12 +52,22 @@ national_resources:
 <!---Please focus on resources that are relevant for the whole country for life sciences--->
 
 ## Introduction 
-<!---General RDM considerations for your country, how to deal with RDM on a national level--->
+This page provides a general overview of national resources and initiatives on research data management (RDM) in Cyprus
```

Comment:
If possible, you can also add public webpages of DM-related services and personnel from libraries at the universities in Cyprus.